multiqc/modules/nonpareil/nonpareil.py

```diff
@@ -0,0 +1,276 @@
+                }
+                data[f"{self.disp_type}_lower"] = {
+                    x: y for x, y in zip(data["x.adj"], list(np.array(data["y.cov"]) - np.array(disp_add)))
+                }
+
+        return json_raw
+
+    def nonpareil_general_stats_table(self):
+        """Take the parsed stats from the nonpareil report and add it to the
+        General Statistics table at the top of the report"""
+
+        headers = {
+            "L": {
+                "title": "Read Length",
+                "min": 0,
+                "suffix": " bps",
+                "scale": "GnBu",
+                "format": "{:,.3f}",
+                "hidden": True,
+            },
+            "AL": {
+                "title": "Adjusted read length",
+                "description": "Adjusted read length (if using kmers)",
+                "min": 0,
+                "scale": "GnBu",
+                "suffix": " bps",
+                "format": "{:,.3f}",
+                "hidden": True,
+            },
+            "R": {
+                "title": "Number of reads",
+                "min": 0,
+                "scale": "GnBu",
+                "format": "{:,.0f}",
+                "hidden": True,
+            },
+            "LR": {
+                "title": "Sequencing effort",
+                "description": "Effective sequencing effort used",
+                "min": 0,
+                "suffix": " Mbps",
+                "scale": "Blues",
+                "format": "{:,.2f}",
+                "hidden": True,
+            },
+            "overlap": {
+                "title": "Read overlap",
+                "description": "Minimum read overlap",
+                "min": 0,
+                "suffix": " bps",
+                "scale": "Blues",
+                "hidden": True,
+                "format": "{:,.0f}",
+            },
+            "log.sample": {
+                "title": "Log sampling",
+                "description": "Multiplier of the log-sampling (or zero if linear)",
+                "max": 1,
+                "min": 0,
+                "scale": "BuGn",
+                "hidden": True,
+                "format": "{:,.2f}",
+            },
+            "kappa": {
+                "title": "Redundancy",
+                "description": "Dataset redundancy",
+                "max": 1,
+                "min": 0,
+                "scale": "RdYlGn-rev",
+                "format": "{:,.2f}",
+            },
+            "C": {
+                "title": "Coverage",
+                "description": "Dataset coverage",
+                "max": 1,
+                "min": 0,
+                "scale": "RdYlGn",
+                "format": "{:,.2f}",
+            },
+            "consistent": {
+                "title": "Consistent Data?",
+                "description": "Is the data sufficient for accurate estimation?",
+                "modify": lambda x: "Yes" if x == 1 else "No",
```

Comment:
If you use `true` and `false` here instead of `Yes` and `No`, MultiQC will nicely format this field
(these global rules are used across all modules: https://github.com/ewels/MultiQC/blob/2be5617c387266646e3eb2fc62ab43ebc6398075/multiqc/utils/config_defaults.yaml#L75-L85)

Reply:
But then it shows up as `0` and `1`. Do you think that is clear?

Reply:
Hmm, I didn't check but if that's so, we should modify how the `table_cond_formatting_rules` work so it shows words and not numbers 🤔

Reply:
So I'll keep it as `true`/false`, ok?

Reply:
Ah, sorry, I didn't mean `True`, but `"true"`, as a string literal. The values listed `table_cond_formatting_rules` in are those that MultiQC will regonize and format as nice green/red labels. As you can see, `"true"` adn `"false"` are listed there.

Reply:
But I'd modify the core MultiQC code so `True` and `False` are recognised as well. And`"yes"` and `"no"` also, for good measure